Formula/civl.rb

```diff
@@ -0,0 +1,22 @@
+
+class Civl < Formula
+  desc "The Concurrency Intermediate Verification Language"
+  homepage "http://vsl.cis.udel.edu/civl/"
+  url "http://vsl.cis.udel.edu/lib/sw/civl/1.7/r3157/release/CIVL-1.7_3157.tgz"
+  sha256 "33fa5b4877fbf494b0a4b426c15d6a165fa3db48a9f97cbf391818d965d2dfb9"
+  depends_on :java => "1.8+"
```

Comment:
Please leave an empty line between the `sha256` line and the first `depends_on`.